test/testunifiedsearchlistmodel.cpp

```diff
@@ -575,13 +575,22 @@ private slots:
             const auto type = model->data(model->index(i), OCC::UnifiedSearchResultsListModel::DataRole::TypeRole);
 
             if (type == OCC::UnifiedSearchResult::Type::Default) {
-                const auto providerId =
+                const QString providerId =
                     model->data(model->index(i), OCC::UnifiedSearchResultsListModel::DataRole::ProviderIdRole)
                         .toString();
+
+                const QString subline = 
+                    model->data(model->index(i), OCC::UnifiedSearchResultsListModel::DataRole::SublineRole)
+                        .toString();
+
+                const QString title = 
+                    model->data(model->index(i), OCC::UnifiedSearchResultsListModel::DataRole::TitleRole)
+                        .toString();
```

Comment:
we want our code to use `auto` always when possible
```suggestion
                const auto providerId =
                    model->data(model->index(i), OCC::UnifiedSearchResultsListModel::DataRole::ProviderIdRole)
                        .toString();

                const auto subline = 
                    model->data(model->index(i), OCC::UnifiedSearchResultsListModel::DataRole::SublineRole)
                        .toString();

                const auto title = 
                    model->data(model->index(i), OCC::UnifiedSearchResultsListModel::DataRole::TitleRole)
                        .toString();
```